### ISSUE TITLE
How can I proceed with the test with personal data

### ISSUE THREAD

**Issue by kyungpyokwon** [SarahwXU/HiSup#17]
Thank you for your research.
How can I put personal satellite image data into the model created after training with inria dataset and crowdai?

I'd appreciate it if you could tell me

**Comment by SarahwXU** [comment]
Please refer to the ''Quickstart with the pretrained model'' in README after a successful installation. One simple command line is needed.

**Comment by kyungpyokwon** [comment]
Thank you for your answer. I have now executed the following command to test the model trained through the crowdai data.

python scripts/test.py --config-file config-files/crowdai_hrnet48.yaml

After doing this, two json files were created in the /outputs/crowdai_hrnet48 directory. I want to get an image with polygons on the result, is there a way?

**Comment by SarahwXU** [comment]
Please follow the exact code to generate visualized polygonal mapping results: 
`python scripts/demo.py --dataset crowdai --img [YOUR_IMAGE_PATH]`

**Comment by kyungpyokwon** [comment]
No, I don't want to use a pre-trained model, but I want to test a new model that I trained. When I use the save_viz function of visualiizer.py in the utils directory to run the test.py file, can't I print the result image of the polygon?

**Comment by kyungpyokwon** [comment]
I would like to test a new model that I trained without using a pre-trained model. In this regard, I am planning to utilize the save_viz function from visualizer.py in the utils directory within the context of the test_pipeline.py file, test_on_crowdai function. However, it seems that I am unable to print the result image of the polygon. Could you kindly assist me with this matter?

**Comment by SarahwXU** [comment]
The visualization mainly depends on the format of the derived results. Results generated by our pretrained model are in the COCO instance segmentation format, which is resistant to the AICrowd dataset. In that format, buildings are delineated by corner points. The simple function is the [show_polygons](https://github.com/SarahwXU/HiSup/blob/69e0cb3c7039bd038139ae0b95ea4ef03b2e5162/hisup/utils/visualizer.py#L32C1-L32C1.). If your predicted results by your model have the same structure as ours, you could try writing the code for visualization based on that function.

**Comment by kyungpyokwon** [comment]
Thank you very much. Successful visualization of the polygon. I would like to evaluate the inria_dataset as well, but the method in the readme seems to be a way to evaluate the crowdai_dataset. How can I evaluate inria_dataset?

**Comment by XJKunnn** [comment]
To evaluate inria dataset, you can follow the [instruction](https://project.inria.fr/aerialimagelabeling/contest/).

**Comment by kyungpyokwon** [comment]
I always appreciate your kind words. I've suddenly encountered the following error while performing multiple model training and testing. If you've ever experienced such an error or have any solutions, I would be grateful if you could let me know.
![화면 캡처 2023-08-10 171041](https://github.com/SarahwXU/HiSup/assets/128347413/0cafc93c-b0a2-4bac-9223-f0c05ba19dc5)

**Comment by SarahwXU** [comment]
Regrettably, we haven't encountered this error. The problem seems to be the checkpoint loader. Maybe you could try solely debugging the loading of checkpoints instead of running the whole test file.

**Comment by SarahwXU** [comment]
Regrettably, we haven't encountered this error. The problem seems to be the checkpoint loader. Maybe you could try solely debugging the loading of checkpoints instead of running the whole test file.
